CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+
+The documentation for each repository is available in the "**docs/**" directory of that repository.
+It is written in [Markdown format] and built using [MkDocs].
+
+## Testing Your Changes Locally
+
+We **strongly** encourage you to test your changes locally before contributing them.
+To do so, you first need to do three things; these are:
+
+1. Have MkDocs installed.
+2. Have the Laminas documentation theme available to the laminas-form directory.
+3. Update `mkdocs.yml` to build the docs using the documentation theme.
+
+To complete these steps, please follow the instructions below.
+
+1. [Install MkDocs].
+2. Clone the [documentation-theme repository] locally.
+    ```
+    git clone git@github.com:laminas/documentation-theme.git documentation-theme
+    ```
+3. Copy the theme directory from the cloned documentation-theme repository to the root directory of this project.
+4. Replace lines 91 and 92 of `mkdocs.yml` with the following Yaml snippet, changing them to suit the repository that you are working with.
```

Comment:
Not sure if mentioning line number is a good idea. They change over time and most likely without updating this document. 
Maybe we can add a "comment section" in that file which we can refer too.

Reply:
> They change over time and most likely without updating this document.

It is different for each repository. Examples:

* https://github.com/laminas/laminas-mvc/blob/e3c420a96349f57655ed28b135290e331edb579c/mkdocs.yml#L26-L27
* https://github.com/laminas/laminas-session/blob/6486dbcebfc80e2c128f2da522b19a68d24b5cfa/mkdocs.yml#L15-L16
* https://github.com/laminas/laminas-session/blob/6486dbcebfc80e2c128f2da522b19a68d24b5cfa/mkdocs.yml#L15-L16
* https://github.com/laminas/laminas-view/blob/2e961a1f698593f8db6a71c193e3312783b46b68/mkdocs.yml#L42-L43

And changed after a new page is added to the documentation. Example: https://github.com/laminas/laminas-session/pull/29

Reply:
Thanks for spotting that. I'll get that section updated.

Reply:
I updated the documentation to make it more generic, and applicable to any of the repos.